dev-db/mydumper/mydumper-0.10.7.2.ebuild

```
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_P="${PN}-$(ver_rs 3 -)"
+
+DESCRIPTION="A high-performance multi-threaded backup (and restore) toolset for MySQL"
+HOMEPAGE="https://github.com/maxbube/mydumper"
+SRC_URI="https://github.com/maxbube/${PN}/archive/v${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+RDEPEND="app-arch/zstd
+	dev-db/mysql-connector-c:=
+	dev-libs/glib:=
+	dev-libs/libpcre:=
+	dev-libs/openssl:0=
```

General rule of thumb for adding := is when dependency has a non-zero subslot defined (albeit there are exceptions, like qt deps are special), and not do it otherwise.

And when a package define a non-zero slot (like glib), should also either request it directly or use `:*` if there is multiple slots and they are all known to work.
```suggestion
RDEPEND="app-arch/zstd:=
	dev-db/mysql-connector-c:=
	dev-libs/glib:2
	dev-libs/libpcre
	dev-libs/openssl:=
```
As for openssl, its slots are a thing of a distant past.. not that cleaning the 0 really matters so you'll still see a lot of ebuilds with it.